src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -125,7 +127,19 @@ def create_launcher(fluent_launch_mode: LaunchMode = None, **kwargs):
         return SlurmLauncher(**kwargs)
 
 
+def _show_gui_to_ui_mode_converter_func(old_arg, new_arg, kwargs):
+    warnings.warn(
+        "'show_gui' is deprecated. Use 'ui_mode' instead.",
+        PyFluentDeprecationWarning,
+    )
+    if kwargs[old_arg] is True and kwargs[new_arg] is None:
+        kwargs[new_arg] = UIMode.GUI
+
+
 #   pylint: disable=unused-argument
+@deprecate_arguments(
```

Comment:
@prmukherj Can you please check if the docstring is changed?

Reply:
Thank you Mainak, fixed it.